packages/common-utils/src/types.ts

```diff
@@ -203,8 +203,12 @@ export const AlertIntervalSchema = z.union([
 
 export type AlertInterval = z.infer<typeof AlertIntervalSchema>;
 
+export const zAlertChannelType = z.literal('webhook');
```

Comment:
we need to add a changeset to this package (for every changes)

Reply:
added changeset for common-utils